testgrid/config.yaml

```diff
@@ -3707,6 +3709,13 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 8
+    num_failures_to_alert: 4 # Runs every 2h. Alert when it's been
```

Comment:
If I remember well, now the definition of the kubeadm jobs is sorted by job type (before kind jobs, then upgrade jobs, then X on Y)
If so, please move the external-etcd jobs definition down

Reply:
Done, at the end of the `# kubeadm-kinder tests` block.